ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/PartyMapper.java

```diff
@@ -75,29 +77,36 @@ public Party from(CCDParty ccdParty) {
             case COMPANY:
                 CCDCompany ccdCompany = ccdParty.getCompany();
                 return new Company(ccdCompany.getName(), addressMapper.from(ccdCompany.getAddress()),
-                    addressMapper.from(ccdCompany.getCorrespondenceAddress()),
-                    ccdCompany.getPhoneNumber(), representativeMapper.from(ccdCompany.getRepresentative()),
-                    ccdCompany.getContactPerson());
+                    addressMapper.from(ccdCompany.getCorrespondenceAddress()), ccdCompany.getPhoneNumber(),
+                    representativeMapper.from(ccdCompany.getRepresentative()), ccdCompany.getContactPerson());
             case INDIVIDUAL:
                 CCDIndividual ccdIndividual = ccdParty.getIndividual();
                 return new Individual(ccdIndividual.getName(), addressMapper.from(ccdIndividual.getAddress()),
                     addressMapper.from(ccdIndividual.getCorrespondenceAddress()), ccdIndividual.getPhoneNumber(),
                     representativeMapper.from(ccdIndividual.getRepresentative()),
-                    LocalDate.parse(ccdIndividual.getDateOfBirth(), DateTimeFormatter.ISO_DATE));
+                    parseDob(ccdIndividual.getDateOfBirth()));
             case SOLE_TRADER:
                 CCDSoleTrader ccdSoleTrader = ccdParty.getSoleTrader();
                 return new SoleTrader(ccdSoleTrader.getName(), addressMapper.from(ccdSoleTrader.getAddress()),
-                    addressMapper.from(ccdSoleTrader.getCorrespondenceAddress()), ccdSoleTrader.getPhoneNumber(),
-                    representativeMapper.from(ccdSoleTrader.getRepresentative()), ccdSoleTrader.getTitle(),
-                    ccdSoleTrader.getBusinessName());
+                    addressMapper.from(ccdSoleTrader.getCorrespondenceAddress()),
+                    ccdSoleTrader.getPhoneNumber(), representativeMapper.from(ccdSoleTrader.getRepresentative()),
+                    ccdSoleTrader.getTitle(), ccdSoleTrader.getBusinessName());
             case ORGANISATION:
                 CCDOrganisation ccdOrganisation = ccdParty.getOrganisation();
                 return new Organisation(ccdOrganisation.getName(), addressMapper.from(ccdOrganisation.getAddress()),
-                    addressMapper.from(ccdOrganisation.getCorrespondenceAddress()), ccdOrganisation.getPhoneNumber(),
-                    representativeMapper.from(ccdOrganisation.getRepresentative()), ccdOrganisation.getContactPerson(),
-                    ccdOrganisation.getCompaniesHouseNumber());
+                    addressMapper.from(ccdOrganisation.getCorrespondenceAddress()),
+                    ccdOrganisation.getPhoneNumber(), representativeMapper.from(ccdOrganisation.getRepresentative()),
+                    ccdOrganisation.getContactPerson(), ccdOrganisation.getCompaniesHouseNumber());
             default:
                 throw new MappingException();
         }
     }
+
+    private LocalDate parseDob(String dateOfBirth) {
```

Comment:
Since this is the only parse method in this class I think we can call it just `parse` and have more concise `parse(ccdIndividual.getDateOfBirth()))`. What do you think?